sensor/kubernetes/listener/resources/convert.go

```diff
@@ -194,6 +195,11 @@ func (w *deploymentWrap) populateNonStaticFields(obj interface{}, action *centra
 		podLabels = o.Spec.JobTemplate.Spec.Template.GetLabels()
 		podSpec = o.Spec.JobTemplate.Spec.Template.Spec
 		labelSelector = o.Spec.JobTemplate.Spec.Selector
+	case *batchv1.CronJob:
```

Comment:
I assume it's safe to remove the case block above (`*v1beta1.CronJob`). With the handler being registered using `V1()` informer there won't be any events of that type coming in.

Reply:
+1

Reply:
+1

Reply:
Done